### PR TITLE
feat: xp command strings

### PR DESCRIPTION
**Pull request by FnrDev**
**Please describe the changes this PR makes and why it should be merged:**

**In this PR:**

### CODE REVIEW

File: en/bot.json [hunk 1]
```diff
@@ -482,5 +482,16 @@
 	"vip_whitelist_description": "Whitelist a user to allow access to the ProBot panel.",
 	"vip_whitelist_user_description": "The user to add / remove from ProBot panel.",
 	"vip_no_perms": ":x: You need administrator permission to use vip commands.",
-  "button_someone_else": ":x: This button belongs to someone else."
-}
\ No newline at end of file
+  	"button_someone_else": ":x: This button belongs to someone else.",
+	"xp_description": "Add / remove xp from a user.",
```
Comment (Dramex) [inline en/bot.json:486]:
```suggestion
	"setxp_description": "Add / remove xp from a user.",
```

File: en/website.json [hunk 2]
```diff
@@ -731,5 +731,6 @@
     "temp_max_description": "The number of channels a user may create at once.",
     "temp_channel_module": "Temporary Channels",
     "rr_max_reactions": "Max reactions reached!",
-    "temp_max_users": "Default maximum number of users"
+    "temp_max_users": "Default maximum number of users",
+    "xp_description": "Add / remove xp from a user."
```
Comment (Dramex) [inline en/website.json:735]:
```suggestion
    "setxp_description": "Changes member xp."
```

**Review by Dramex — CHANGES_REQUESTED** [review]
fixes

File: en/bot.json [hunk 3]
```diff
@@ -493,5 +493,6 @@
 	"setxp_changed_xp": "**✅ Changed xp for @{0}**",
 	"setxp_select_type": "**Please select type of xp.**",
 	"setxp_text_select": "Adds xp in text",
```
Comment (Anasqx) [inline en/bot.json:495]: Adds xp to the text level

**Review by Anasqx — CHANGES_REQUESTED** [review]

File: en/bot.json [hunk 4]
```diff
@@ -483,15 +483,15 @@
...
-	"xp_remove_description": "Remove xp from a user",
-	"xp_remove_user_description": "The user you want to remove xp from",
-	"xp_remove_xp_description": "The number of removing xp from the user.",
-	"xp_changed_xp": "**✅ Changed xp for @{0}**",
-	"xp_select_type": "**Please select type of xp.**",
-	"xp_text_select": "Adds xp in text",
-	"xp_voice_select": "Adds xp in voice"
+	"setxp_description": "Add / remove xp from a user.",
+	"setxp_add_description": "Add xp to user.",
+	"setxp_add_user_description": "The user you want to add xp to.",
```
Comment (Anasqx) [inline en/bot.json:488]: The user to whom you want to add xp.

File: en/bot.json [hunk 5]
```diff
@@ -483,15 +483,15 @@
...
+	"setxp_add_xp_description": "The number of xp to add to the user.",
```
Comment (Anasqx) [inline en/bot.json:489]: The amount of xp to add to the user.

File: en/bot.json [hunk 6]
```diff
@@ -483,15 +483,15 @@
...
+	"setxp_remove_description": "Remove xp from a user",
+	"setxp_remove_user_description": "The user you want to remove xp from",
+	"setxp_remove_xp_description": "The number of removing xp from the user.",
```
Comment (Anasqx) [inline en/bot.json:492]: The amount of xp you want to remove from the user

File: en/bot.json [hunk 7]
```diff
@@ -483,15 +483,15 @@
...
-	"xp_changed_xp": "**✅ Changed xp for @{0}**",
-	"xp_select_type": "**Please select type of xp.**",
-	"xp_text_select": "Adds xp in text",
-	"xp_voice_select": "Adds xp in voice"
+	"setxp_description": "Add / remove xp from a user.",
+	"setxp_add_description": "Add xp to user.",
+	"setxp_add_user_description": "The user you want to add xp to.",
+	"setxp_add_xp_description": "The number of xp to add to the user.",
+	"setxp_remove_description": "Remove xp from a user",
+	"setxp_remove_user_description": "The user you want to remove xp from",
```
Comment (Anasqx) [inline en/bot.json:491]: The user from whom you want to remove xp.

File: en/bot.json [hunk 8]
```diff
@@ -483,15 +483,15 @@
...
+	"setxp_remove_xp_description": "The number of removing xp from the user.",
+	"setxp_changed_xp": "**✅ Changed xp for @{0}**",
```
Comment (Anasqx) [inline en/bot.json:493]: **☑️ | XP has been updated for @{0}**"

File: en/bot.json [hunk 9]
```diff
@@ -483,15 +483,15 @@
...
+	"setxp_select_type": "**Please select type of xp.**",
```
Comment (Anasqx) [inline en/bot.json:494]: Please select the xp type.

**Review by Anasqx — CHANGES_REQUESTED** [review]

**Pull request merged**